src/main/java/org/mapstruct/intellij/util/MapstructAnnotationUtils.java

```diff
@@ -370,4 +375,41 @@ public static PsiModifierListOwner findMapperConfigReference(PsiAnnotation mappe
 
         return (PsiModifierListOwner) resolvedElement;
     }
+
+    /**
+     * Find the other mapper types used by the class or interface defined in the {@code mapperAnnotation}
+     *
+     * @param mapperAnnotation the mapper annotation in which the mapper config is defined
+     *
+     * @return the classes / interfaces that are defined with the {@code uses} attribute,
+     * or {@code null} if there isn't anything defined
+     */
+    public static List<PsiClass> resolveUsesConfigClasses(PsiAnnotation mapperAnnotation) {
```

Comment:
Can we also look into the `Mapper#config` class for this? We do something with it in `TargetUtils#isBuilderEnabled` (as a reference example). Would also be good to add a test case for it as well